docs/examples/ssh/SSH_BITBUCKET.md

```diff
@@ -0,0 +1,24 @@
+# SSH service in Bitbucket on Kubernetes
+
+In addition to providing a service on HTTP(S), Bitbucket also allows remote Git operations over SSH connections. By default, Kubernetes Ingress controllers only work for HTTP connections, but some ingress controllers also support TCP connections.
+
+You can provide SSH access by creating a separate Kubernetes `LoadBalancer` service, but you can do this only if you don’t need your deployment to have the SSH service available on the same DNS name as the HTTP service.
+
+## Creating a separate Kubernetes LoadBalancer service for SSH connections on AWS
+In the values file for the helm chart, the extra SSH service can be enabled like this:
+```
+bitbucket:
+  sshService:
+    enabled: true
+```
+On a deployment using AWS, assuming you have [external-dns](https://github.com/kubernetes-sigs/external-dns) configured, you can add these annotations to automatically set up the DNS name for the SSH service:
```

Comment:
It is not clear if this is optional or not. I would say if we are talking about having a different domain name for the ssh service then this is not optional or if we make `external-dns` optional, then we need to also say that if you don't have it, you need to create DNS configuration for the newly created loadbalancer.

Reply:
Yep also a good call - `external-dns` just helps with setting up the DNS name, but you can also do it manually of course